presto-main/src/main/java/com/facebook/presto/operator/PartitionedLookupSource.java

```diff
@@ -112,15 +118,15 @@ public int getChannelCount()
     @Override
     public long getJoinPositionCount()
     {
-        return Arrays.stream(lookupSources)
```

Comment:
Not sure this should be statically imported (since there are also `Streams.stream` one could want to use), but if you decide to keep static import, add it in a separate commit.

Reply:
Agreed, I was too aggressive with the change. I'll revert it.